app/views/builds.html

```diff
@@ -49,11 +49,11 @@
                     <th ng-class="{'hidden-sm' : (latestByConfig | hashSize)}">Source</th>
                   </tr>
                 </thead>
-                <tbody ng-if="!(latestByConfig | hashSize)">
+                <tbody ng-if="!(latestByConfig | hashSize) && !(buildsNoConfig | hashSize)">
                   <tr><td colspan="7"><em>{{emptyMessage}}</em></td></tr>
                 </tbody>
-                <tbody ng-if="(latestByConfig | hashSize)">
-                  <tr ng-repeat="(buildConfigName, latestBuild) in latestByConfig">
+                <tbody ng-if="(latestByConfig | hashSize) || (buildsNoConfig | hashSize)">
+                  <tr ng-repeat="(buildConfigName, latestBuild) in latestByConfig track by (latestBuild | uid)" ng-if="buildConfigName !== ''">
```

Comment:
hmm good point, do we actually even need track by in this case then? will ng-repeat track by the key for a hash? looking...